clients/js/src/provider.ts

```diff
@@ -80,10 +82,10 @@ export function isWrappedEthereumProvider<P extends EIP2696_EthereumProvider>(
  * @param options (optional) Re-use parameters from other providers
  * @returns Sapphire wrapped provider
  */
-export function wrapEthereumProvider<P extends EIP2696_EthereumProvider>(
+export async function wrapEthereumProvider<P extends EIP2696_EthereumProvider>(
```

Comment:
Does it affect all integrations? For example I don't see ether-v6 is relying on `wrapEthereumProvider`. It has it's own logic.